### ISSUE TITLE
Additional questions for doc on fit classifier

### ISSUE THREAD

**Issue by rsenft1** [piximi/documentation#6]
These are some questions I think we should cover in our description of the fit classifier: 

How do I pick what optimization algorithm to use? 

How do I pick a loss function? 

How do I select batch size / learning rate / # epochs? (What happens if this is too big, what happens if too small, and how to determine what a reasonable range is?) 

What should the learning curve look like? 

What does it mean if my accuracy is wildly fluctuating during training? / flat / increases monotonically? (generally, how do we interpret this shape?)

Essentially, I think what we need is a flow chart of sorts to guide scientists through training a model and evaluating that training. There are a bunch of hyperparameters to change and we should include some guidance about 1) which circumstances (training performance, type of data, metrics under 'evaluate' etc.) should trigger tweaking and 2) what direction to move in (if possible to know) and 3) how much of a change to make (should batch size be incremented by 1 or 10? I think we should also include some guidance over which algorithms and loss functions to start with.
